sktime/utils/datetime.py

```diff
@@ -61,15 +61,20 @@ def _coerce_duration_to_int(duration, freq=None):
 
 
 def _get_freq(x):
-    """Get unit for conversion of time deltas to integers"""
+    """Get unit for conversion of time deltas to integer."""
     if hasattr(x, "freqstr"):
-        return x.freqstr
+        if x.freqstr is None:
```

Comment:
> Note that `freqstr` has also become deprecated now, not sure if changing this goes beyond the scope of the current PR.

line 72: if 'freqstr' doesn't exist, then it will return None.
Also, I assume the setup.py file will specify the pandas version. so I assume if Sktime is installed with specified pandas version, hopefully, it should be OK for the user to use.

Reply:
Let me know whether this makes sense.

Reply:
I'd say it's beyond scope, but we should open an issue.
What would the "fix" for use of `freqstr` be?